packages/package.go

```diff
@@ -92,6 +92,7 @@ type BasePolicyTemplate struct {
 	Icons           []Image          `config:"icons,omitempty" json:"icons,omitempty" yaml:"icons,omitempty"`
 	Categories      []string         `config:"categories,omitempty" json:"categories,omitempty" yaml:"categories,omitempty"`
 	DeploymentModes *DeploymentModes `config:"deployment_modes,omitempty" json:"deployment_modes,omitempty" yaml:"deployment_modes,omitempty"`
+	BaseDataStreams []string         `config:"data_streams,omitempty" json:"data_streams,omitempty" yaml:"data_streams,omitempty"`
```

Comment:
Nit. Do we need the `Base` prefix here? I think it is needed in `BasePackage` because this type is embedded into `Package`, that also has `DataStreams`.
```suggestion
	DataStreams []string         `config:"data_streams,omitempty" json:"data_streams,omitempty" yaml:"data_streams,omitempty"`
```

Reply:
👍 will make the change